src/ResourceManager/SubscriptionDefinition/Commands.SubscriptionDefinition/help/Get-AzureRmSubscriptionDefinition.md

```diff
@@ -0,0 +1,66 @@
+---
+external help file: Microsoft.Azure.Commands.SubscriptionDefinition.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSubscriptionDefinition
+
+## SYNOPSIS
+Get a subscription definition.
+
+## SYNTAX
+
+### Get subscription definitions.
+```
+Get-AzureRmSubscriptionDefinition
+```
+
+## EXAMPLES
+
+### Example 1
```

Comment:
@wilcobmsft for your examples, would you mind providing the resulting output as well? This will give users an idea of what is returned

Reply:
Done.